e2e/support/helpers/api/createDashboardWithQuestions.ts

```diff
@@ -16,14 +16,11 @@ export const createDashboardWithQuestions = ({
   dashboardName?: string;
   dashboardDetails?: DashboardDetails;
   questions: (NativeQuestionDetails | StructuredQuestionDetails)[];
-  cards?: Partial<Card>[];
-}): Cypress.Chainable<
-  Cypress.Response<{
-    dashboard: Dashboard;
-    questions: Card;
-  }>
-> => {
-  // @ts-expect-error - Cypress typings don't account for what happens in then() here
+  cards?: Partial<DashboardCard>[];
```

Comment:
types were wrong. I used this function inside `e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts`. Without this change, TypeScript would complain